opba-banking-rest-api/src/main/resources/static/tpp_banking_api_ais.yml

```diff
@@ -39,6 +39,7 @@ paths:
         - $ref: "./tpp_banking_api_commons.yml#/components/parameters/PSU-Consent-Session"
         - $ref: "./tpp_banking_api_commons.yml#/components/parameters/Service-Session-ID"
         - $ref: "./tpp_banking_api_commons.yml#/components/parameters/X-Request-ID"
+        - $ref: "./tpp_banking_api_commons.yml#/components/parameters/Use-Cache"
```

Comment:
`online=false` is more understandable compared to `Use-OPB-Cache`. Please rename to `online` and the `online` parameter is by default `true`. If false results are read from cache